third_party/move/move-compiler-v2/tests/checking/inlining/lambda_cast_err.exp

```diff
@@ -0,0 +1,7 @@
+
+Diagnostics:
+error: invalid call of `vector::fold`: expected `integer` but found `vector<u8>` for argument 3
```

Comment:
At least we have an error, but can you file a medium-priority bug to revisit this message?  The relationship between `vector<u8>` and `argument 3` is pretty indirect.